tools/python_tools/cuvslam_tools/tests/test_dataset_preparation_cli.py

```
@@ -51,6 +52,175 @@ def test_prepare_euroc_installed_defaults(self):
+            )
+            curl.chmod(0o755)
+            md5sum = fake_bin / "md5sum"
+            md5sum.write_text(
+                "#!/usr/bin/env bash\n"
+                "printf '363f5c2502b469cdd97ef85997714806  %s\\n' \"$2\"\n"
+            )
+            md5sum.chmod(0o755)
+            env = dict(os.environ)
+            env["PATH"] = f"{fake_bin}:{env['PATH']}"
+
+            completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)